app/views/settings/index.html.erb

```diff
@@ -1,7 +1,8 @@
 <% title _("Settings") %>
 <% title_actions documentation_button('3.5.2ConfigurationOptions') %>
 <ul class="nav nav-tabs" data-tabs="tabs">
-  <% @settings.group_by(&:category).each do |category, setting| %>
+    <% @settings.group_by(&:category).each do |category, setting| %>
+    <% next if setting.blank? %>
```

Comment:
I'm a bit confused by this, is it supposed to hide remnant 'puppet' settings that are blank? Under what circumstances would a setting be blank here?